faust/stores/rocksdb.py

```diff
@@ -400,7 +400,7 @@ def _dbs_for_key(self, key: bytes) -> Iterable[DB]:
 
     def _dbs_for_actives(self) -> Iterator[DB]:
         actives = self.app.assignor.assigned_actives()
-        topic = self.table._changelog_topic_name()
+        topic = self.table.changelog_topic_name
```

Comment:
Isn't this a method name? I don't see a property with that name

Reply:
It's defined as a property at https://github.com/faust-streaming/faust/blob/be1e6dbe2507fd068a40391dcd2ddd436372661e/faust/tables/base.py#L608